app/controllers/api/v1/followers_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::FollowersController < ApplicationController
+  def create
+    @follower = Follower.create(user_id: params[:user_id], follower_user: current_user)
+
+    if @follower.save
+      render json: { status: 'Success', message: 'You have successfully followed' }, status: 201
+    else
+      render json: { status: 'Error', message: @follower.errors.full_messages }, status: 422
+    end
+  end
+
+  private
+
+  def follower_params
+    params.require(:follower).permit(:user_id)
+  end
+end
```

Comment:
Hello son, I believe you forgot to create a unfollow method...
- To do this, you can access the current_user followings (since we only want authenticated users to be able to follow other users) and then check if the id of the user he wants to follow is already in there, if it is in there, then you can create a destroy method, otherwise, a create method...
`
@following_users = current_user.followings

@isFollow = @following_users.exists?(following_user_id: params[:id])

if @isFollow
//codes to unfollow

else
//code to follow
end
`